docs/helpers/WebDriver.md

```diff
@@ -1094,7 +1094,7 @@ Returns **[Promise][25]&lt;[string][18]>** attribute value
 ### moveCursorTo
 
 Moves cursor to element matched by locator.
-Extra shift can be set with offsetX and offsetY options.
+Extra shift can be set with xOffset and yOffset options.
```

Comment:
Did you just updated .MD doc, without  editing mustache template or JSdoc of WebDriver method?

Is it will be overwritten in any next build of doc?

Reply:
only WebDriver has this change. So I don't update the mustache template.